sites/www.thunderbird.net/about/index.html

```diff
@@ -122,6 +122,7 @@ <h2 aria-label="{{ _('Governed by Contributors') }}">{{ _('Governed by <span cla
             this council is elected by contributors and acts as their voice in determining Thunderbird's strategic
             direction and goals. The Council oversees funds, approves budgets, and shapes the product roadmap,
             ensuring Thunderbird remains true to its core values.
+	   <a class="strong"  href="{{ url('council.thunderbird.net') }}">{{ _('Learn more about the Thunderbird Council') }}</a>	
```

Comment:
Thanks for the PR. So first of all, this link should not be part of the <p> tag. It should be below the closing `</p>`. Otherwise it doesn't match the other links of this type on the page.

Secondly, 

The url function is Python and pulls the link from `URL_MAPPINGS` in https://github.com/thunderbird/thunderbird-website/blob/master/settings.py#L167 -- it exists as a shortcut for longer links and to make it simple to change a link when necessary.

Given this is a single link not likely to be reused anywhere else anytime soon, you don't need to use it, `https://council.thunderbird.net` is fine.